src/Akeneo/Pim/Structure/Family/back/Infrastructure/Query/Sql/SqlCountFamilyCodes.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace Akeneo\Pim\Structure\Family\Infrastructure\Query\Sql;
```

Comment:
Missing also elsewhere in other classes in this PR, but is this still relevant?

```suggestion

declare(strict_types=1);

namespace Akeneo\Pim\Structure\Family\Infrastructure\Query\Sql;
```